packages/keystone/file/file-types/excel.js

```diff
@@ -28,67 +27,20 @@ class ExcelParser {
         return true
     }
 
-    /**
-     * Excel stores date types as numbers <days from epoch>.<time data>
-     * @param serial {number} date from excel parser
-     * @returns {string} YYYY-MM-DD or YYYY-MM-DDTHH:MM:SS
+    /** @param {excel.CellObject} cell
+     *  @returns {string | undefined}
      */
-    static parseExcelDate (serial) {
-        const withTime = String(serial).includes('.')
-
-        // milliseconds since 1899-12-31T00:00:00Z, corresponds to Excel serial 0.
-        const xlSerialOffset = -2209075200000
-
-        let elapsedDays
-        // each serial up to 60 corresponds to a valid calendar date.
-        // serial 60 is 1900-02-29. This date does not exist on the calendar.
-        // we choose to interpret serial 60 (as well as 61) both as 1900-03-01
-        // so, if the serial is 61 or over, we have to subtract 1.
-        if (serial < 61) {
-            elapsedDays = serial
-        }
-        else {
-            elapsedDays = serial - 1
+    parseCell (cell) {
```

Comment:
If cell contains number '0', it will be in cell.v.l
And I'm here just checking, that cell exists.
So for cell with number '0', number '0' will be returned
Same with 'false'